docker/test/requirements.txt

```diff
@@ -0,0 +1,6 @@
+confluent_kafka
```

Comment:
This doesn't seem right.

Reply:
It seems like it's a typo. Updating it to `confluent-kafka`

Reply:
But why would we depend on a confluent kafka artifact.

Reply:
@ijuma Confluent Kafka for Python - https://github.com/confluentinc/confluent-kafka-python
`confluent-kafka` is a python client compatible with kafka
For sanity tests of docker images we wanted a python client to interact with kafka residing in docker image, hence we went ahead with this publicly available and widely used python client.
We are open to consider alternate python clients if there are any concerns in using this library

Reply:
Can we not use the CLI tools that are included in the Apache Kafka project?

Reply:
This suggestion has been incorporated. Now we are using CLI scripts included in kafka.

Reply:
Can we remove `confluent_kafka` from here then?

Reply:
It has been removed. These changes are part of KIP 975 PR raised here:- https://github.com/apache/kafka/pull/14552. (commit 6f64ce54254b985a50b3c5d8d63711cc6b71ab18)
They'll reflect here once this PR is updated

Reply:
Makes sense, thanks.